SharedContent/cpp-winui-template/WinUI3TemplateCpp/SampleConfiguration.cpp

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include "pch.h"
+#include "SampleConfiguration.h"
+#include "MainPage.xaml.h" 
+
+using namespace winrt;
+using namespace winrt::WinUI3TemplateCpp;
+using namespace Windows::Foundation::Collections;
+
+IVector<Scenario> implementation::MainPage::scenariosInner = winrt::single_threaded_observable_vector<Scenario>(
+	{
+		Scenario{ L"Scenario 1", L"WinUI3TemplateCpp.Scenario1_ShortName"},
```

Comment:
```suggestion
		Scenario{ L"Scenario 1", winrt::name_of<WinUI3TemplateCpp::Scenario1_ShortName>() },
```

Use `name_of` so that refactoring will update the page name when the class is renamed.

Reply:
changed to using `name_of`